rebar.config

```diff
@@ -18,11 +18,18 @@
     {layout, edoc_layout_chunks},
     {preprocess, true},
     {dir, "_build/default/lib/hex_core/doc"}]}.
+{project_plugins, [
```

Comment:
The plugin will not be pulled in for consumers of `hex_core` (this is expected).